v21.1/alembic.md

```diff
@@ -0,0 +1,569 @@
+---
+title: Migrate CockroachDB Schemas with Alembic
+summary: Learn how to use Alembic with a CockroachDB cluster.
+toc: true
+---
+
+This page walks you through a series of simple database schema changes using the [Alembic](https://alembic.sqlalchemy.org/en/latest/) schema migration module with a simple Python application built on SQLAlchemy and CockroachDB.
+
+For a detailed tutorial about using Alembic, see the [the Alembic documentation site](https://alembic.sqlalchemy.org/en/latest/tutorial.html).
+
+For information about specific migration tasks, see Alembic's [Cookbook](https://alembic.sqlalchemy.org/en/latest/cookbook.html).
+
+## Before you begin
+
+Before you begin the tutorial, [install CockroachDB](install-cockroachdb.html).
+
+## Step 1. Start a cluster and create a database
```

Comment:
Have we tried using Alembic with CockroachCloud? Might be worth seeing if we could add CC free-tier instructions here eventually.